neuromation/cli/utils.py

```diff
@@ -489,6 +489,25 @@ def parse_resource_for_sharing(uri: str, root: Root) -> URL:
     return uri_from_cli(uri, root.username, allowed_schemes=("storage", "image", "job"))
 
 
+async def parse_and_resolve_resource_for_sharing(uri: str, root: Root) -> URL:
+    """ Parses the neuromation resource URI string + resolves job-name to job-ID.
+    Available schemes: storage, image, job. For image URIs, tags are not allowed.
+    """
+    uri_obj = parse_resource_for_sharing(uri, root)
+    if uri_obj.scheme == "job":
+        job_id = await resolve_job(
+            str(uri_obj), client=root.client, default_user=root.username
+        )
+        uri_obj = uri_obj.with_path(job_id)
+    return uri_obj
+
+
+async def normalize_job_resource(uri: URL, root: Root) -> URL:
```

Comment:
Is it used anywhere?

Reply:
you're right, it's redundant method left from older refactorings! Thank